python/hsfs/core/data_validation_engine.py

```diff
@@ -84,22 +91,37 @@ def validate(self, feature_group, feature_dataframe):
             validation_results_python = []
             for validation_result_java in exp_res.getResults():
                 # Create rule python
+                legal_values = []
+                if validation_result_java.getRule().getLegalValues() is not None:
+                    for (
+                        legal_value
+                    ) in validation_result_java.getRule().getLegalValues():
+                        legal_values.append(legal_value)
```

Comment:
```suggestion
                legal_values = [legal_value for legal_value in validation_result_java.getRule().getLegalValues()] if validation_result_java.getRule().getLegalValues() is not None else []
```
same question as above?